src/utils/scroll-offset.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * This source file is part of the Swift.org open source project
+ *
+ * Copyright (c) 2021 Apple Inc. and the Swift project authors
+ * Licensed under Apache License v2.0 with Runtime Library Exception
+ *
+ * See https://swift.org/LICENSE.txt for license information
+ * See https://swift.org/CONTRIBUTORS.txt for Swift project authors
+*/
+
+import { documentationTopicName } from 'docc-render/constants/router';
+
+export const EXTRA_DOCUMENTATION_OFFSET = 10;
+
+/**
+ * @param {import('vue-router').Route} to
+ * @returns {Number}
+ */
+export default function getExtraScrollOffset(to) {
+  const { name } = to;
+  const isDocumentation = name.includes(documentationTopicName);
```

Comment:
Is it safe to assume that `name` will never be undefined here? I think it is—just double checking.

Reply:
Yep, we always have a name to our routes.